devicemodel/samples/apl-mrb/launch_uos.sh

```diff
@@ -17,8 +17,10 @@ ipu_passthrough=1
 fi
 
 # use the modprobe to force loading snd-soc-skl/sst_bxt_bdf8532
-modprobe -q snd-soc-skl
-modprobe -q snd-soc-sst_bxt_tdf8532
+modprobe -q snd_soc_skl
+modprobe -q snd_soc_tdf8532
+modprobe -q snd_soc_sst_bxt_sos_tdf8532
```

Comment:
@wenlingz  Can you have a check of existence of this module as a hint if it is a new SOS version.

Reply:
@binbinwu1 please help check again,thanks